### ISSUE TITLE
I lost training data when training VOC data format, but the test data was not lost

### ISSUE THREAD

**Issue by eduo97** [SwinTransformer/Swin-Transformer-Object-Detection#122]
During my training, my training data is less than the actual training set, but the data of the verification set and the test set do not change。Has anyone ever had this problem?

**Comment by impiga** [comment]
Hi, @eduo97 . Could you show how many images are used during training and how many images are in the training set?
